modules/ossm-uninstalling-service-mesh-operator-control-plane.adoc

```diff
@@ -0,0 +1,165 @@
+.. In the {ocp-short-name} web console, click **Operators** -> **Installed Operators**.
+.. Click **IstioCNI** in the **Provided APIs** column.
+.. Click the **Options** menu -> **Delete IstioCNI**.
+.. At the prompt to confirm the action, click **Delete**.
+
+. Uninstall {SMProductName} 3 Operator:
+.. In the {ocp-short-name} web console, click **Operators** -> **Installed Operators**.
+.. Locate {SMProductName} 3 Operator.
+.. Click the **Options** menu -> **Uninstall Operator**.
+.. At the prompt to confirm the action, click **Uninstall**.
+
+. Delete the `istio-system` project:
+.. In the {ocp-short-name} web console, click  **Home** -> **Projects**.
+.. Locate the name of the `istio-system` project.
+.. Click the **Options** menu -> **Delete Project**.
+.. At the prompt to confirm the action, enter the name of the project.
+.. Click **Delete**.
+
+. Delete the `istio-cni` project:
+.. In the {ocp-short-name} web console, click  **Home** -> **Projects**.
+.. Locate the name of the `istio-cni` project.
+.. Click the **Options** menu -> **Delete Project**.
+.. At the prompt to confirm the action, enter the name of the project.
+.. Click **Delete**.
+
+If you decide to delete Istio custom resource definitions, then run the following command:
+
+[source, terminal]
+----
+$ oc get crds -oname | grep istio.io | xargs kubectl delete
+----
+
+[NOTE]
+====
+If no resources remain, it is safe to remove the CRD or `APIServices`. Otherwise, you should not delete the type as the wanted resources will be deleted automatically when the CRD or APIService is deleted.
+====
+
+== Uninstall {SMProduct} by using the CLI
+
+. Delete the `Istio` resource:
+
+.. Retrieve the `Istio` resource name by running the following command:
++
+[source, terminal]
+----
+$ oc get istio -n istio-system
+----
+
+.. Replace `<name_of_custom_resource>` with the output from the previous command, and run the following command to remove the custom resource:
++
+[source, terminal]
+----
+$ oc delete istio/<name_of_custom_resource>
+----
+
+. Delete the `IstioCNI` resource:
+
+.. Retrieve the name of the `IstioCNI` resource by running the following command:
++
+[source, terminal]
+----
+$ oc get istioCNI -n istio-system
+----
+
+.. Replace `<name_of_custom_resource>` with the output from the previous command, and run the following command to remove the custom resource:
++
+[source, terminal]
+----
+$ oc delete istioCNI/<name_of_custom_resource>
+----
+
+. Delete the `istio-system` and `istio-cni` Projects:
+
+.. Delete the `istio-system` namespace by running the following command:
++
+[source, terminal]
+----
+$ oc delete ns istio-system
+----
+
+.. Delete the `istio-cni` namespace by running the following command:
++
+[source, terminal]
+----
+$ oc delete ns istio-cni
+----
+
+. Uninstall {SMProduct} Operator 3:
+
+.. Check the current version of the `servicemeshoperator3` subscribed Operator in the `currentCSV` field by running the following command:
++
```

Comment:
@gwynnemonahan : will change the order.

(iii) delete the subscription
(I) get current CSV
(ii) delete the CSV